### DISCUSSION TITLE
Help needed to install Kratix Jenkins promise

### DISCUSSION THREAD

**Discussion by anandreads** [syntasso/kratix#267] [Q&A]
Team, I am trying to install a Kratix jenkins in my Macbook by having 2 clusters (created using Kind) based on the instructions given here in this link https://docs.kratix.io/main/guides/installing-a-promise. But when I tried to run this command "kubectl --context $PLATFORM apply 
  --filename "${KRATIX_MARKETPLACE_REPO}/jenkins/promise.yaml" 

When I clone in my local and run the above the kubectl command, am getting this error **"E1101 13:20:39.816918   60134 memcache.go:265] "Unhandled Error" err="couldn't get current server API group list: Get \"https://127.0.0.1:64120/api?timeout=32s\": net/http: TLS handshake timeout"**

When I use your github repo as by setting this variable KRATIX_MARKETPLACE_REPO="https://github.com/syntasso/kratix-marketplace/tree/main/"  and run the above the kubectl command, am getting this error "error: error parsing https://github.com/syntasso/kratix-marketplace/tree/main/jenkins/promise.yaml: error converting YAML to JSON: yaml: line 216: mapping values are not allowed in this context" 

But the promise.yaml has only 81 lines but why the above error message shows in line # 216. Please help me on this. I am stuck on creating jenkins

**Comment by kirederik** [comment]
Hey @anandreads 

Sorry for the very late reply -- we missed this question.

Do you still need help or did you manage to fix it?
